humanoid_league_interactive_marker/humanoid_league_interactive_marker/interactive_marker.py

```diff
@@ -563,23 +553,25 @@ def publish_marker(self):
 def main(args=None):
     rclpy.init(args=args)
     node = rclpy.create_node(NODE_NAME)
-    server = InteractiveMarkerServer(node, "")
+    server = InteractiveMarkerServer(node, "robocup")
+    multi_executor = MultiThreadedExecutor()
```

Comment:
Why is the `MultiThreadedExecuter` necessary?

Reply:
See the discussion in the ros2 migration group. Essentially, it is a hacky way to receive tf updates while another callback is active.